src/window.rs

```diff
@@ -736,13 +736,17 @@ impl Window {
 
 /// Monitor info functions.
 impl Window {
-    /// Returns the monitor on which the window currently resides
+    /// Returns the monitor on which the window currently resides.
+    ///
+    /// If `None` was returned it could mean that window isn't presented on any monitor.
+    /// However you must not treat it as it's minimized, obscured, or localted on some
+    /// virtual desktop.
```

Comment:
I'd personally rephrase this, since the wording is awkward and it's not particularly useful. Particularly the "**was** returned" and "**could** mean" seems very strange.

I'm also not sure what the note about not treating it as minimized/obscured/localted(?)/virtual desktop is about? Are you trying to say that people should still render to it?

Reply:
>I'm also not sure what the note about not treating it as minimized/obscured/localted(?)/virtual desktop is about? Are you trying to say that people should still render to it?

I'm saying that users must not use that to detect that window is minimized, obscured, or something in their render code to make a decision whether to render or not.

Reply:
You mean because when people never render there would never be a monitor? Seems a bit backwards to phrase it like that.

Reply:
No, I mean that they shouldn't do something like that.

```rust
loop {

  // Reading events. 

  if window.current_monitor.is_none() {
    continue;
  }

  render();
}
```

Reply:
Why not? Assuming that it has been rendered once before?

Reply:
Well, if we're talking about Wayland a canonical way to do that is frame callbacks, since you might be asked to render your window when minimized. So purely confusing. I can remove this doc all together, saying that on Wayland it can return None.

Reply:
I think that would be better, yeah. I feel like this documentation was made in good faith, but will likely just lead to confusion.

Reply:
I've updated the docs, let me know what you think.